locator/tablets.cc

```diff
@@ -474,27 +485,30 @@ class tablet_effective_replication_map : public effective_replication_map {
     inet_address_vector_replica_set to_replica_set(const tablet_replica_set& replicas) const {
         inet_address_vector_replica_set result;
         result.reserve(replicas.size());
+        auto& topo = _tmptr->get_topology();
+        for (auto&& replica : replicas) {
+            auto* node = topo.find_node(replica.host);
+            if (node) {
+                result.emplace_back(node->endpoint());
+            }
```

Comment:
that's essentially token_metadata::get_endpoint_for_host_id_if_known(replica.host)
but there's no problem open-coding it here.
Eventually, we may want to return `node_vector_replica_set` and use the endpoint addresses only as the last step for communicating to them.